src/idex/identity-resolver.js

```diff
@@ -2,10 +2,20 @@ import { toParams } from '../utils/url'
 import { fromError } from '../utils/emitter'
 import { expiresInHours, asParamOrEmpty, asStringParam, mapAsParams } from '../utils/types'
 import { DEFAULT_IDEX_EXPIRATION_HOURS, DEFAULT_IDEX_AJAX_TIMEOUT, DEFAULT_IDEX_URL } from '../utils/consts'
+import { base64UrlEncode } from '../utils/b64'
 
 const IDEX_STORAGE_KEY = '__li_idex_cache'
 
-function _responseReceived (storageHandler, domain, expirationHours, successCallback) {
+function _cacheKey (additionalParams) {
```

Comment:
this is the only thing that is needed to make the cache work for different params.
Everything else is fixing the linting and making the storage handler generic.